Packs/GoogleChronicleBackstory/Integrations/GoogleChronicleBackstory/GoogleChronicleBackstory.py

```diff
@@ -197,37 +214,84 @@ def __init__(self, params: Dict[str, Any], proxy, disable_ssl):
         else:
             self.region = REGIONS['General']
 
+    def _implement_retry(self, retries=0,
+                         status_list_to_retry=None,
+                         backoff_factor=5,
+                         raise_on_redirect=False,
+                         raise_on_status=False):
+        """
+        Implements the retry mechanism.
+        In the default case where retries = 0 the request will fail on the first time.
 
-''' HELPER FUNCTIONS '''
+        :type retries: ``int``
+        :param retries: How many retries should be made in case of a failure. when set to '0'- will fail on the first time.
 
```

Comment:
I see that this method is copied from the baseClient class in the commonServerPython file. Could we just inherit it? (class Client(BaseClient):)

Please feel free to reach out to me with any questions
I'm available here or on slack.

Reply:
The reason for not using BaseClient was to create the authorized session through the Google Auth2 library, where BaseClient creates its own session which is not required here.